README.md

```diff
@@ -123,7 +123,7 @@ $yform->setValueField('text', ['postalcode','PLZ','','0']);
 $yform->setValueField('text', ['city','Ort','','0']);
 $yform->setValueField('number', ['lat','LAT','10','7','','0','input:text']);
 $yform->setValueField('number', ['lng','LNG','11','8','','0','input:text']);
-$yform->setValueField('osm_geocode', ['osm','OSM','lat,lng','street,postalcode,city','500','','0']);
+$yform->setValueField('osm_geocode', ['osm','OSM','lat,lng','street,postalcode,city','500','','','0']);
```

Comment:
BC: Reihenfolge der Parameter ändert sich, Anmerkung dazu von mir im Value.

Reply:
Ist m.E. unkritisch, denn den auch der letzte Parameter ist gestern oder vorgestern ert von mir eingebaut worden und nicht in einem Release veröffentlicht. Den kann noch niemand gentzt haben, daher kein BC.